src/panel.css

```diff
@@ -88,13 +88,31 @@ h3 {
   font-size: 14px;
 }
 
-.Facebook-text::after { /* grey Facebook with fence */
-  background-image: url("/img/fbc-icon2.svg");
+.Facebook-text::after {
+  background: url("/img/fbc-icon2.svg") center no-repeat;
+  background-size: 14px auto;
   display: inline-block;
   width: 14px;
   height: 12px;
   margin-left: 4px;
   margin-bottom: -1px;
+  content: "";
+}
+
+.Facebook-blue-text {
+  color: var(--secondaryText);
```

Comment:
Needs to be `--blue60` or `--blue70` to make it blue. Otherwise it's the same gray is 67+, which doesn't match the container icon color:

<img width="342" alt="image" src="https://user-images.githubusercontent.com/71928/57820666-55623080-7753-11e9-87b0-8f9f2ea30108.png">